vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -147,9 +148,14 @@ vic_cesm_put_data()
 
                 // albedo: direct , visible
                 // CESM units: unitless
-                albedo = AreaFactor *
-                         (atmos->shortwave[NR] /
-                          (atmos->shortwave[NR] - energy.NetShortAtmos));
+                if (atmos->shortwave[NR] > 0.) {
+                    albedo = AreaFactor *
+                             (atmos->shortwave[NR] /
```

Comment:
This is confusing. Is `atmos->shortwave[NR]` the incoming shortwave or the net shortwave. Either the variable naming is confusing or the equation is wrong. Provide some comments to explain what you are doing here (including sign conventions).